addons/hr_timesheet/controllers/portal.py

```diff
@@ -181,7 +179,7 @@ def get_timesheets():
             'search': search,
             'sortby': sortby,
             'groupby': groupby,
-            'searchbar_inputs': searchbar_inputs,
+            'searchbar_inputs': self._get_searchbar_inputs(),
```

Comment:
Why this change ?

Reply:
Because in another file, I was asked to do this change so I did it in every similar file. Will remove this one then.